src/sentry/seer/similarity/grouping_records.py

```diff
@@ -61,7 +62,7 @@ def post_bulk_grouping_records(
 
     try:
         response = make_signed_seer_api_request(
-            seer_grouping_connection_pool,
+            seer_grouping_backfill_connection_pool,
```

Comment:
Do you actually want to use the backfill pool here? Or keep it on `seer_grouping_connection_pool`?

Reply:
the backfill is not currently used.

the other url has been switched to the good one.